subsys/net/lib/dhcpv4/Kconfig

```diff
@@ -85,6 +85,17 @@ config NET_DHCPV4_SERVER_ADDR_LEASE_TIME
 	  The lease time determines when the IPv4 address lease expires if the
 	  client does not renew it.
 
+config NET_DHCPV4_SERVER_ADDR_DECLINE_TIME
+	int "The time IPv4 addresses remains blocked after conflict detection (seconds)"
+	default 86400
+	help
```

Comment:
Would it make sense to add a range for allowable values?
We do not have ranges for other int values in this kconfig either which might be a bit problematic especially as we do not check the value in the code.

Reply:
I just wonder what would be correct range values, it's not specified by RFC so we'd need to choose some arbitrary ones. That's why I've left it w/o range.

Reply:
I was mostly concerned if user sets timeout as negative value, and that value is fed directly to `K_SECONDS()`, what is the timeout value in that case. This is of course quite academic, why would anyone put <0 value here.
Anyway, I was just thinking what could go wrong. We could limit the value to [0, MAX_INT] for example.

Reply:
Good point with the negative values. Ok, I've applied ranges for the time options, for lease (and decline) time uint32_t max value, as it's the theoretical maximum that lease time option can carry. For probe timeout, 60 seconds max, the maximum is already high enough to make the server replies sluggish, I don't think setting anything higher would make sense at all.